src/middlewared/middlewared/etc_files/ntp.conf

```diff
@@ -0,0 +1,27 @@
+<%
+	ntp_query = middleware.call_sync('system.ntpserver.query')
+%>\
+% for ntp in ntp_query:
+server ${ntp['address']}\
+	% if ntp['burst']:
+ burst\
+	% endif
+	% if ntp['iburst']:
+ iburst\
+	% endif
+	% if ntp['prefer']:
+ prefer\
+	% endif
+	% if ntp['maxpoll']:
+ maxpoll ${ntp['maxpoll']}\
+	% endif
+	% if ntp['minpoll']:
+ minpoll ${ntp['minpoll']}\
+	% endif
+
+% endfor
+restrict default limited kod nomodify notrap nopeer noquery
+restrict -6 default limited kod nomodify notrap nopeer noquery
+restrict 127.0.0.1
+restrict -6 ::1
+restrict 127.127.1.0
```

Comment:
From what I understand there restrict lines should only be written if there are ntp servers?